### ISSUE TITLE
JavaScript blocked icon does not appear on v1.19.86

### ISSUE THREAD

**Issue by ugultopu** [brave/brave-browser#13693]
## Description 
I block JavaScript by default. On Brave, this is done through "Settings -> Shields -> Block scripts". Blocking the scripts still work (that is, JavaScript will be blocked by default) _but_ the little icon that shows on the address bar saying that JavaScript is blocked will not show up. 

## Steps to Reproduce

   1. Go to "Settings -> Shields" and enable "Block scripts".
   2. Go to any page that the JavaScript has not been whitelisted.
   3. Observe that the JavaScript is blocked, _but_ the icon that says that JavaScript is blocked does not appear on the address bar.

## Expected result: Icon appears.
Just like it does on the previous versions of Brave, and current and previous versions of Chrome:
![image](https://user-images.githubusercontent.com/9937444/105255793-55364d00-5b52-11eb-9686-a09608be82c9.png)

## Other Additional Information: Does not happen with Chrome 88.

**Comment by MikaelPorttila** [comment]
Connected to this change https://github.com/brave/brave-browser/issues/199

**Comment by rebron** [comment]
Closing as wontfix. Intended change via #199
